### ISSUE TITLE
ApplicationRevision.spec.referredObjects  array should have exactly one sub-item

### ISSUE THREAD

**Issue by justxi1** [kubevela/kubevela#3874]
**Describe the bug**
get error when apply application
```txt
SchemaError(dev.oam.core.v1beta1.ApplicationRevision.spec.referredObjects): array should have exactly one sub-item） 
```
**To Reproduce**

yaml 
```yaml
apiVersion: core.oam.dev/v1beta1
kind: Application
metadata:
  name: cluster-app
spec:
  components:
    - name: simple-test
      type: webservice
      properties:
        image: nginx
```
I uninstall the kubevela v1.1.11 and reinstall kubevela v1.3.4 
I have execute command to clear CRD before reinstall 
```bash
kubectl get crd |grep oam | awk '{print $1}' | xargs kubectl delete crd
```
**Expected behavior**
**Screenshots**
![image](https://user-images.githubusercontent.com/39458411/168021971-d20d3588-221d-4a52-b286-92bb2dd74d19.png)

**KubeVela Version**

CLI Version: v1.3.4
Core Version: 1.3.4
GitRevision: git-d748096
GolangVersion: go1.17.9

**Cluster information**

kubernet v1.19.2

**Additional context**

**Comment by Somefive** [comment]
This could be a potential problem for k8s 1.19. Try to run the following command and see if it can help.
```bash
kubectl patch crd applicationrevisions.core.oam.dev --type json -p='[{"op": "remove", "path": "/spec/versions/1/schema/openAPIV3Schema/properties/spec/properties/referredObjects/x-kubernetes-preserve-unknown-fields"}]'
```

**Comment by Somefive** [comment]
Seems should be fixed after k8s v1.19.5. https://github.com/kubernetes/kubernetes/pull/96562

**Comment by justxi1** [comment]
> This could be a potential problem for k8s 1.19. Try to run the following command and see if it can help.
> 
> ```shell
> kubectl patch crd applicationrevisions.core.oam.dev --type json -p='[{"op": "remove", "path": "/spec/versions/1/schema/openAPIV3Schema/properties/spec/properties/referredObjects/x-kubernetes-preserve-unknown-fields"}]'
> ```

thanks, it work
